go.mod

```diff
@@ -29,3 +29,5 @@ require (
 	gopkg.in/ini.v1 v1.42.0 // indirect
 	gopkg.in/yaml.v2 v2.2.2 // indirect
 )
+
+go 1.13
```

Comment:
I thought we were staying on 1.12.  Are we not? (I don't have any firm view either way. Just asking in case this commit was in error)

Reply:
Sorry, go mod keeps adding it. I've read through the 1.13 release notes, and they look mostly fine. I still think we should stick to 1.12 for now, and upgrade to 1.13 in the next release.

The build seems to work fine, with Go version 1.12 and such a go.mod file with "go.13".